keyboards/plain60/keymaps/yanfali/README.md

```diff
@@ -0,0 +1,9 @@
+- Disable VIA
```

Comment:
Would you please rename this as `readme.md`?

Reply:
Sure. But itsa GitHub convention. They use this name themselves when you init a repo.

Reply:
https://github.com/yanfali/nametest

Reply:
Noroads kindly explained to me that this is really to prevent issues with case insensitive file systems like HFS and NTFS where git checkouts with readme.md and README.md will cause git to fail in hard to predict ways. As a cross platform project we gotta fight for the users.